angular-workspace/projects/ni/nimble-angular/src/directives/select/testing/select.pageobject.ts

```diff
@@ -0,0 +1,3 @@
+import { SelectPageObject } from '@ni/nimble-components/dist/esm/select/testing/select.pageobject';
```

Comment:
We want to export page objects from a separate entry point so that clients can enforce that they only be used from test code, not production code. 

The way we've done this for table and rich text is to create an entry point for the component and an entry point for the page object. If we follow that pattern we'd have to move the select which would be a breaking change. We want to do this eventually but maybe now's not the time?

An alternative might be to create the select page object in the correct long term location but leave the select where it is for now. But I'm not sure if that'll cause problems if things in the main nimble-angular entry point try to use it.

Reply:
I was able to create the `select/testing` entry point and use the page object from the entry point in the existing select control value accessor test.